roles/installer/templates/statefulsets/postgres.yaml.j2

```
@@ -46,6 +46,29 @@ spec:
 {% endif %}
 {% if postgres_priority_class is defined %}
       priorityClassName: '{{ postgres_priority_class }}'
+{% endif %}
+{% if init_postgres_extra_commands %}
```

This is causing error if `init_postgres_extra_commands` is not used in AWX CR:

```bash
 TASK [Create Database if no database is specified] ******************************** 
fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'init_postgres_extra_commands' is undefined. 'init_postgres_extra_commands' is undefined. 'init_postgres_extra_commands' is undefined. 'init_postgres_extra_commands' is undefined\n\nThe error appears to be in '/opt/ansible/roles/installer/tasks/database_configuration.yml': line 186, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n- block:\n    - name: Create Database if no database is specified\n      ^ here\n"}
```

We should use `is defined` for this conditional.

```suggestion
{% if init_postgres_extra_commands is defined %}
```

---

Thanks!  I added this and your other code suggestions.